src/sidebar/components/annotation-header.js

```diff
@@ -34,14 +36,16 @@ export default function AnnotationHeader({
   // NB: `created` and `updated` are strings, not `Date`s
   const hasBeenEdited =
     annotation.updated && annotation.created !== annotation.updated;
-  const showTimestamp = !isEditing;
+  const showTimestamp = !isEditing && annotation.created;
```

Comment:
This little "fix" avoids the possibility of (in tests, at least), an attempt to render a `Timestamp` without a `created` property; this will raise Errors in props validation in tests.